src/Config.php

```diff
@@ -74,6 +74,9 @@ protected function configureOptions(OptionsResolver $resolver)
             'constructorParamsDefaultToNull' => false,
             'soapClientClass'               => '\SoapClient',
             'soapClientOptions'             => array(),
+            'soapServerClass'               => '\SoapServer',
+            'soapServerOptions'             => array(),
+            'serverClassName'               => '',
```

Comment:
Is there any reason to provide a configuration option for setting the server class name? We do not do this for the client class name and the developer will have to subclass it anyway allowing her to choose the name of the subclass freely.